var/spack/repos/builtin/packages/jdk/package.py

```diff
@@ -55,6 +55,8 @@ class Jdk(Package):
     # found in a link above. The build number can be deciphered from the URL.
     # Alternatively, run `bin/java -version` after extracting. Replace '+'
     # symbol in version with '_', otherwise it will be interpreted as a variant
+    version('10.0.2_13', sha256='6633c20d53c50c20835364d0f3e172e0cbbce78fff81867488f22a6298fa372b', curl_options=curl_options,
```

Comment:
technically you don't need the `sha256=`, spack infers the type from the hash length